tests/adapters_t/sites_t/test_moab.py

```diff
@@ -216,6 +261,43 @@ def test_resource_status(self):
         del expected_resource_attributes.updated, return_resource_attributes.updated
         self.assertEqual(return_resource_attributes, expected_resource_attributes)
 
+    @mock_executor_run_command(TEST_RESOURCE_STATE_TRANSLATION_RESPONSE)
+    def test_resource_state_translation(self):
```

Comment:
Consider to [parameterize](https://docs.pytest.org/en/latest/parametrize.html) this test so that failing statuses are individually marked. See [cobald's test_weighted](https://github.com/MatterMiners/cobald/blob/master/cobald_tests/composite/test_weighted.py#L88) as an example.

```suggestion

    @pytest.mark.parametrize("state", TEST_RESOURCE_STATE_STATUS)
    @mock_executor_run_command(TEST_RESOURCE_STATE_TRANSLATION_RESPONSE)
    def test_resource_state_translation(self):
```

Reply:
We are using the unittest framework, so `pytest.mark.parametrize` has probably no effect here, right?

Reply:
Ah, my bad. CoffeeOverflowException.